app/api/routes/image_queries.py

```diff
@@ -54,6 +55,45 @@ async def validate_query_params_for_edge(request: Request, invalid_edge_params:
         )
 
 
+def handle_iqe_creation(
```

Comment:
Abstracted this code into a helper because otherwise it would have been duplicated. Functionality should be the same.

Reply:
Nice - we have a bunch of helper functions at the bottom of this file, can we colocate this with them, and also prepend an underscore to indicate that its a private helper function?